### ISSUE TITLE
Connection pool size parameters ignored?

### ISSUE THREAD

**Issue by michaelhixson** [redstone-dart/redstone_mapper_pg#1]
In this snippet of code, in lib/manager.dart, aren't the min and max constructor parameters for PostgreSqlManager being ignored?

https://github.com/redstone-dart/redstone_mapper_pg/blob/247822a019bb894097dd1646209932c0233e4141/lib/manager.dart#L24

```
  PostgreSqlManager(String uri, {int min: 1, int max: 3}) {
    _pool = new Pool(uri, min: 1, max: 3); // <-- Should this be min:min, max:max?
    _init = _pool.start();
  }
```

**Comment by azenla** [comment]
@michaelhixson Good Catch. In case you aren't aware, the ownership of redstone has changed, so the new team is in the learning phase of this code base. I'll push an update later today.
